src/mixins/referencesProvider.js

```diff
@@ -21,8 +22,39 @@ export default {
       }),
     },
   },
+  data: () => ({ appState: AppStore.state }),
   computed: {
     // exposes the references for the current page
-    references: ({ store }) => store.state.references,
+    references() {
+      const {
+        isFromIncludedArchive,
+        store: {
+          state: { references: originalRefs = {} },
+        },
+      } = this;
+      // if present, use `includedArchiveIdentifiers` data to determine which
+      // references should still be considered active or not
+      return Object.keys(originalRefs).reduce((newRefs, id) => ({
+        ...newRefs,
+        [id]: {
+          ...originalRefs[id],
+          isFromIncludedArchive: isFromIncludedArchive(id),
+        },
+      }), {});
```

Comment:
I'm not passing the identifiers, just adding a new boolean flag that is derived from a computation using them.

I can look into removing the URL instead of adding the flag, although I expect the code to look pretty similar since `Reference` always expects a URL, and I need to update the places where it is used today to possibly handle situations without a URL.

Reply:
Ok. I changed it to just strip out `url` fields for the inactive references. I had to update `Reference` to handle an optional URL, so I'll need to do a little more testing to make sure that doesn't break things.